src/transformers/trainer_utils.py

```diff
@@ -46,7 +46,27 @@
     import tensorflow as tf
 
 
-def set_seed(seed: int):
+def seed_worker():
+    worker_seed = torch.initial_seed() % 2**32
+    set_seed(worker_seed)
+
+
+def set_deterministic_seed_for_cuda():
+    #  Enable PyTorch deterministic mode. This potentially requires either the environment
```

Comment:
Function missing docstring. This docstring could also contain a link to the [reproducibility page of the PyTorch docs](https://pytorch.org/docs/stable/notes/randomness.html).